mne_bids/write.py

```diff
@@ -851,7 +851,8 @@ def write_raw_bids(raw, bids_path, events_data=None,
             participants.tsv
             scans.tsv
 
-        Note that the datatype ``'meg'`` is automatically inferred from the raw
+        Note that ``bids_path`` will be modified in place as well as returned
+        and that the datatype ``'meg'`` is automatically inferred from the raw
```

Comment:
write_raw_bids does not only right fif files. it can also write EEG. I feel this needs to be rephrased.